examples/blueprint-construct/index.ts

```diff
@@ -71,6 +70,13 @@ export default class BlueprintConstruct extends Construct {
             new blueprints.addons.KubeviousAddOn(),
             new blueprints.addons.EbsCsiDriverAddOn(),
             new blueprints.addons.EfsCsiDriverAddOn({replicaCount: 1}),
+            new blueprints.addons.KedaAddOn({
```

Comment:
Is it mutually exclusive with the MetricsServer? If not, we can keep it. Ideally, I would like to have a warning when both are deployed, but we can defer this.

Reply:
It wont conflict with each other.. Keda creates the metrics server in "keda" namespace..